ras_backstage/controllers/sample_controller.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from structlog import wrap_logger
+
+from ras_backstage import app
+from ras_backstage.common.requests_handler import request_handler
+from ras_backstage.exception.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def upload_sample(survey_id, collection_exercise_id, sample_file, survey_type='B'):
```

Comment:
Issue found in sample service, which incorrectly expects a survey type of ["bres" / "social" / "census"](https://github.com/ONSdigital/rm-sample-service/blob/da3b122b3696e64c28af872c38f720af2f6337df/src/main/java/uk/gov/ons/ctp/response/sample/service/impl/SampleServiceImpl.java#L234) and not "B". The service also does not raise an error upon [receipt of an unexpected type](https://github.com/ONSdigital/rm-sample-service/blob/da3b122b3696e64c28af872c38f720af2f6337df/src/main/java/uk/gov/ons/ctp/response/sample/service/impl/SampleServiceImpl.java#L245) and still returns an empty 201.